packages/ember-glimmer/tests/integration/components/closure-components-test.js

```diff
@@ -330,7 +331,7 @@ moduleFor('@htmlbars Components test: closure components', class extends Renderi
     this.assertText('Hodi Sigmundur 33');
   }
 
-  ['@glimmer bound outer named parameters get updated in the right scope']() {
+  ['@skip bound outer named parameters get updated in the right scope']() {
```

Comment:
I made a mistake in #13906, didn't realize the entire module was being skipped for `@glimmer`. We should see if we can fix this issue, but currently it does not work in either engine from my understanding.